frontend/ui/web/index.template.html

```diff
@@ -18,6 +18,8 @@
   <link rel="manifest" href="manifest.json">
 </head>
 <body>
+  <!-- this div is for instantiating the PlacesService object to get place details -->
+  <div id="maps"></div>
```

Comment:
I'm not sure if this is what we want (seems weird to have a hard-coded maps container on _every_ view). However, I don't have a better solution. Make sure to run this by @KoopaKing.

Reply:
I mean, you can dynamically insert the node, but that can get tricky too so I'd just as soon not do it.